redisCacheModule.js

```diff
@@ -277,6 +292,12 @@ function redisCacheModule(config){
     if(arguments.length < 1){
       throw new Exception('INCORRECT_ARGUMENT_EXCEPTION', '.del() requires a minimum of 1 argument.');
     }
+    if(keys === 'object') {
```

Comment:
Did you mean `if(typeof keys === 'object') {`?

Reply:
ooops

Reply:
:)